### ISSUE TITLE
Central Metadata Registry

### ISSUE THREAD

**Issue by MKonopko** [ga4gh/TASC#1]
Per M Haendel:

I think we need a central, computable metadata registry for approved standards. This information can then be automatically ingested by other systems. For example, we have a really nice metadata registry for the OBO ontology standards here. Here is an example for HPO: https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/ontology/hp.md

**Comment by jb-adams** [comment]
@MKonopko @mellybelly great idea, I think this is similar to something I had in mind, essentially a registry and API for GA4GH standards so they can be looked up in a programmatic way. I wrote a [document](https://github.com/ga4gh/standards-schema/blob/master/PROPOSAL.md) outlining some use cases for such an application.

I plan to bring this up as a project proposal once the TASC meetings start.

**Comment by mcupak** [comment]
@jb-adams Very nice! I wonder if we can repurpose the Service Registry format for this.

**Comment by jb-adams** [comment]
@mcupak I think this would integrate well with Service Registry and Service Info. For example, each item returned from `/services/types` could be passed to the "Standards API," letting the registry master node access the OpenAPI spec and informing them on what client requests each service could be expected to handle. Could be useful for networks of diverse service types.

**Comment by mamanambiya** [comment]
@jb-adams this sounds like this would be more useful for external use (GA4GH standards consumers) than for GA4GH internal use, right?

**Comment by mellybelly** [comment]
yes, this ensures that journals and other standards registries can easily ingest the released GA4GH standards metadata and recommend them ;-).

**Comment by jb-adams** [comment]
yes, I'm largely seeing it as a way for newcomers and other consumers to explore our available standards, and be able to locate all of our outputs from a central resource or service.

**Comment by ajhpage** [comment]
@mellybelly and @MKonopko - we think this issue is resolved through the release of service registry and service info. If that's not the case, please let us know. see: https://github.com/ga4gh-discovery/ga4gh-service-registry
